docs/section22/Rule22-42.md

```diff
@@ -0,0 +1,99 @@
+
+# CHW & CW - Rule 22-42  
+
+**Rule ID:** 22-42   
+**Rule Description:** The sets of performance curves specified in Table J-2 should be used to represent part-load performance of chillers in the baseline building design.  
+**Rule Assertion:** Baseline RMD = expected value  
+**Appendix G Section:** G3.2.2.1 Baseline 
+
+**Mandatory Rule:** True  
+**Evaluation Context:** Each Chiller  
+**Table Lookup:**
+J-6
+**Function Call:**  
+
+## Applicability Check:  
+- look at each chiller - only water-cooled chillers are applicable (ie chillers with a condensing_loop): `for chiller in B_RMD.chillers:`
+    - if the chiller has a condensing_loop, and the chiller energy source is electricity, continue to rule logic: `if chiller.consensing_loop != NULL and chiller.energy_source_type == "ELECTRICITY": CONTINUE TO RULE LOGIC`
+
+        ## Rule Logic:
+        - figure out the set of performance curves needed for the chiller - this should be V, X, Y, Z, AA or AB.  Initialize a variable curve_set and set it to "NONE": `curve_set = "NONE"`
+        - get the chiller rated capacity: `rated_capacity = chiller.rated_capacity`
+        - convert the chiller rated capacity from watts to tons: `chiller_capacity_tons = rated_capacity * 0.000284345`
+        - check if the compressor type is CENTRIFUGAL: `if chiller.compressor_type == "CENTRIFUGAL":`
+            - if the capacity is less than 150 tons, the category is Z: `if chiller_capacity_tons < 150: curve_set = "Z"`
+            - else if the capacity is greater than or equal to 300 tons, the category is AB: `elif chiller_capacity_tons >= 300: curve_set = "AB"`
```

Comment:
probably also need to add what the category is if the capacity = 150 tons, is that category AA or Z?

Reply:
this is covered in the following line.  Anything that's not less < 150 and not >= 300 (ie 150 - 300, including 150, but not 300)

Reply:
The way you wrote it made me think twice about this too. Since the point of RDS is clear documentation I would recommend writing it so that the case where capacity = 150 tons is not implicitly covered by the `else` condition

Reply:
updated